templates/project/.gitattributes.twig

```diff
@@ -6,3 +6,7 @@
 *.md export-ignore
 {{ tests_path }} export-ignore
 {{ docs_path }} export-ignore
+Makefile export-ignore
+phpunit.xml.dist export-ignore
```

Comment:
This is correct? There are two `.dist` files and one normal file that should not be exported

Reply:
These are the files I found in my vendors folder and that seemed unnecessary to me.